docs/source/existing_code.rst

```diff
@@ -114,8 +114,8 @@ A simple CI script could look something like this:
 
 .. code-block:: text
 
-    python3 -m pip install mypy==0.600  # Pinned version avoids surprises
-    scripts/mypy  # Runs with the correct options
+    python3 -m pip install mypy==0.790  # Pinned version avoids surprises
```

Comment:
Would be nice if we can automatically render the most recent release here, but probably not worth the effort